lms/djangoapps/verify_student/utils.py

```diff
@@ -53,6 +53,25 @@ def earliest_allowed_verification_date():
     return now() - datetime.timedelta(days=days_good_for)
 
 
+def relevant_verifications(candidates, deadline):
```

Comment:
Can we rename this to something like `active_verifications`? I don't think it's clear what relevant means without looking into this function.

Reply:
Updated